### DISCUSSION TITLE
Create /docs folder with Wiki content

### DISCUSSION THREAD

**Discussion by Kongkille** [electricitymaps/electricitymaps-contrib#3052] [Ideas]
Currently, the Wiki content is located in the MD files of the wiki. Right now everyone has edit rights, meaning anyone can make "silent" changes. 

I think keeping the wiki, but moving the MD files to a /docs folder is better and just linking to the seperate docs (as we do with data sources) would be better because:

- When coding you can jump to the /docs folder to see instructions/guides
- Easier to review and maintain the docs as all changes have to go through a PR
- More motivating to make changes to the docs when you get a review IMO
- We can still keep the Wiki, but just have links to the separate docs folders, so you still get the Wiki feeling and we are able to set up different titles, groups etc
- I've seen a /docs folders quite a lot and I've liked it quite a bit

**Comment by FelixDQ** [comment]
What about using a GH action like that syncs the wiki with a /docs folder? I haven't tried it before, but maybe something like this works https://github.com/marketplace/actions/bi-directional-wiki-sync-action

**Reply by Kongkille** [reply]
Uh yeah, that actually seems super smart
